lib/apps/trello.ts

```diff
@@ -0,0 +1,14 @@
+import { findPattern } from "@/lib/findPattern";
+import { FixedStatus, type AppMeta } from "../../types";
+
+export const Trello: AppMeta = {
+    icon: "https://bxp-content-static.prod.public.atl-paas.net/img/favicon.ico",
+    id: "trello",
+    friendlyName: "Trello",
+    twitter: "trello",
+    async checkIsFixed() {
+      return FixedStatus.NOT_FIXED
```

Comment:
You just hardcoded it to say not fixed

Reply:
That is right. I couldn't find a URL where i can check if it is fixed. trello is in app store and i have no idea how to get a download url for a app store hosted app.